### ISSUE TITLE
Add support for user_info field in URL field-set

### ISSUE THREAD

**Issue by vinit-elastic** [elastic/ecs#2123]
Whenever the [uri_parts](https://www.elastic.co/guide/en/elasticsearch/reference/current/uri-parts-processor.html) processor is used, it produces many fields out of which most of the fields are part of the [URL](https://www.elastic.co/guide/en/ecs/master/ecs-url.html) ecs field-set. However, the field user_info is not one of them. Therefore, to avoid wrong mappings and conflict in the mappings, this issue here is to request an enhancement in the URL ecs field-set to include the user_info field.

**Comment by ebeahan** [comment]
Good catch @vinit-elastic - I agree it makes sense to include `user_info` into the `url.*` field set alongside the rest of the `uri_parts` produced fields.
